src/ui-components/custom-overflow-menu/custom-items/add-past-visit.tsx

```diff
@@ -1,16 +1,18 @@
 import React from "react";
-import CustomOverflowMenuItem from "../overflow-menu-item.component";
 import { useTranslation } from "react-i18next";
 import { openVisitDashboard } from "../../../widgets/visit/visit-button.component";
+import { ExtensionSlot } from "@openmrs/esm-framework";
 
 export default function AddPastVisitOverflowMenuItem() {
   const { t } = useTranslation();
   return (
-    <CustomOverflowMenuItem
-      itemText={t("Add Past Visit", "Add Past Visit")}
-      onClick={() =>
-        openVisitDashboard(`${t("visitDashboard", "Visit Dashboard")}`)
-      }
+    <ExtensionSlot
```

Comment:
Hm I am not 100% sure what you are trying to do here.

It seems that you want a very particular item to appear here. That's not what this should be.